en/advanced/gimbal_control.md

```diff
@@ -85,21 +67,23 @@ make px4_sitl gazebo_typhoon_h480
 
 To just test the mount driver on other models or simulators, make sure the driver runs (using `vmount start`), then configure its parameters.
 
-
 ## Testing
-The driver provides a simple test command - it needs to be stopped first with `vmount stop`.
+
+The driver provides a simple test command — it needs to be stopped first with `vmount stop`.
 The following describes testing in SITL, but the commands also work on a real device.
 
 Start the simulation with (no parameter needs to be changed for that):
+
 ```
 make px4_sitl gazebo_typhoon_h480
 ```
+
 Make sure it's armed, eg. with `commander takeoff`, then use the following command to control the gimbal (for example):
+
 ```
 vmount test yaw 30
```

Comment:
Does this still work.

Reply:
1. This should still work.
2. That would be nice, if you feel like trying it out.

Reply:
Not really, at this time :-).